iroha/src/query.rs

```diff
@@ -32,11 +32,14 @@ pub struct VerifiedQueryRequest {
 /// This trait should be implemented for all Iroha Queries.
 #[allow(clippy::missing_errors_doc)]
 pub trait Query {
+    /// Output type of query
+    type Output: Into<Value> + TryFrom<Value>;
+
     /// Execute query on the `WorldStateView`.
     /// Should not mutate `WorldStateView`!
     ///
     /// Returns Ok(QueryResult) if succeeded and Err(String) if failed.
-    fn execute(&self, world_state_view: &WorldStateView) -> Result<Value>;
+    fn execute(&self, world_state_view: &WorldStateView) -> Result<Self::Output>;
 }
```

Comment:
Can we add `execute_into_value` or something like this, not to map all the time? https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=ee6d6a79678631c185e00a099fc92002

Reply:
It is works like that in the end anyway.

Reply:
I mean not to use `.execute().map(Into::into)` but write `execute_into_value` instead. One method seems better and is also easier to refactor

Reply:
But it is not a major point, so proceed as you think is better